### ISSUE TITLE
Unsupported operation: IOClient isn't supported on this platform.

### ISSUE THREAD

**Issue by Zeliret** [iamsalnikov/vkapi.dart#1]
Я использую AngularDart, а библиотека судя по всему обращается к апи через dart:io. 
Возможно ли переделать/дополнить обращение к апи через dart:http, чтобы бибилотеку можно было бы использовать в браузере?

**Comment by iamsalnikov** [comment]
Да, я тестировал библиотеку только на серверной стороне. На клиенте было протестировано только получение токена. Напрямую я не подключаю dart:io, но, думаю, что его использует [пакет http](https://github.com/iamsalnikov/vkapi.dart/blob/master/pubspec.yaml#L7), который является зависимостью проекта.

В ближайшую неделю, думаю, я не смогу найти время для того, чтобы решить эту проблему, но постараюсь сделать это как можно быстрее. Если у Вас уже есть решение, то можете сделать PR.
